minecraft_servers/matherion/manifest.json

```diff
@@ -6,11 +6,14 @@
     "%.matherion.eu"
   ],
   "social": {
-    "web": "https://matherion.eu/",
+    "web": "https://matherion.eu",
+    "web-shop": "https://matherion.craftingstore.net",
```

Comment:
Please use the correct key here: `web-shop` => `web_shop`
=> [Documentation reference](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#available-keys)